edge-functions/rewrite-to-external-url/README.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Rewriting at the edge to an external URL
+slug: rewrite-to-external-url
+description: Learn how to rewrite a Next.js app to an external URL using Vercel Edge Functions
+framework: Next.js
+useCase: Edge Functions
+css: Tailwind
+deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/rewrite-to-external-url&project-name=rewrite-to-external-url&repository-name=rewrite-to-external-url
+demoUrl: https://support-site.vercel.pub/
+---
+
+# Rewrite to an External URL
+
+This example shows how to rewrite a Next.js app to an external URL using Next.js middleware & Vercel Edge Functions.
+
+## Demo
+
+https://support-site.vercel.pub/
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+### One-Click Deploy
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
```

Comment:
Can we update the utm from `next-example` to `vercel-example`? There is a PR from Lee updating those in batch thats why its not updated yet 😓

Reply:
Fixing this now!